src/neo/Network/P2P/Peer.cs

```diff
@@ -226,6 +226,7 @@ private void OnTcpConnected(IPEndPoint remote, IPEndPoint local)
                 Context.Watch(connection);
                 Sender.Tell(new Tcp.Register(connection));
                 ConnectedPeers.TryAdd(connection, remote);
+                connection.Tell(new RemoteNode.SendVersion());
```

Comment:
`RemoteNode.SendVersion` should not be sent in `Peer`.

Reply:
@erikzhang I have removed it to LocalNode.